src/main/resources/application.properties

```diff
@@ -1,12 +1,20 @@
 server.port=9096
 spring.application.name = demo service application name
-spring.application.id= 10
+spring.application.id = 10
 # Local mongodb configuration.
-spring.data.mongodb.host=localhost
-spring.data.mongodb.port=27017
-spring.data.mongodb.database=test
+spring.data.mongodb.host = localhost
+spring.data.mongodb.port = 27017
+spring.data.mongodb.database = test
 
 # Logging configuration.
-logging.level.com.assignment.springboot.mongo=DEBUG
-logging.pattern.console= %d{yyyy-MM-dd HH:mm:ss} - %msg%n
-spring.data.customerServiceGetUrl = http://localhost:8090/api/customer/
\ No newline at end of file
+logging.level.com.assignment.springboot.mongo = DEBUG
+logging.pattern.console = %d{yyyy-MM-dd HH:mm:ss} - %msg%n
+spring.data.zuulservice = zuul-service
+spring.url.customerService = /api/customer/
+
+#Eureka Registration
+eureka.client.serviceUrl.defaultZone = http://localhost:8761/eureka/
+spring.application.name = product-service
+eureka.instance.instanceId = ${spring.application.name}:${random.value}
+zuul-gateway.ribbon.eureka.enabled = false
```

Comment:
wait a second.... why is this false ? is there any specific reason ?

Reply:
no need of this property , so removed.